Simplified.xcodeproj/project.pbxproj

```diff
@@ -856,12 +859,33 @@
 /* End PBXContainerItemProxy section */
 
 /* Begin PBXCopyFilesBuildPhase section */
+		17DFC5F5251E84C5003A19CC /* CopyFiles */ = {
+			isa = PBXCopyFilesBuildPhase;
+			buildActionMask = 2147483647;
+			dstPath = "";
+			dstSubfolderSpec = 10;
+			files = (
+				17DFC5F6251E84CF003A19CC /* AudioEngine.xcframework in CopyFiles */,
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+		};
+		17DFC5F7251E863A003A19CC /* CopyFiles */ = {
```

Comment:
again just making sure this is done for SimplyE, SimplyECardCreator, Open eBooks targets.

Reply:
Yeah all three targets are set to `Embed & Sign` and have the `Copy File` in its `Build Phase`